docs/kotlin.md

```diff
@@ -47,3 +48,9 @@ Add Kotlin imports
 apply plugin: "kotlin-android"
 apply plugin: "kotlin-android-extensions"
 ```
+
+3. Open AndroidManifest.xml and add the following line activity:
+
+```
+android:exported="true"
```

Comment:
Are you sure this is necessary? I'm not sure it is (also not sure what it does?).